docs/TestCoverage.md

```diff
@@ -2694,7 +2694,46 @@ expect(node, inputs=[x], outputs=[y],
 
 
 ### DequantizeLinear
-There are 1 test cases, listed as following:
+There are 2 test cases, listed as following:
+<details>
+<summary>dequantizelinear_channels</summary>
+
+```python
+node = onnx.helper.make_node('DequantizeLinear',
+    inputs=['x', 'x_scale', 'x_zero_point'],
+    outputs=['y'],)
+
+# 1-D tensor zero point and scale of size C
+x = np.array([[[[  3,  89],
+                [ 34, 200],
+                [ 74,  59]],
+
+               [[  5,  24],
+                [ 24,  87],
+                [ 32,  13]],
+
+               [[245,  99],
+                [  4, 142],
+                [121, 102]],],], dtype=np.uint8)
+x_scale = np.array([2, 4, 5], dtype=np.float32)
+x_zero_point = np.array([84, 24, 196], dtype=np.uint8)
+y = np.array([[[[-162,   10],
```

Comment:
Instead of adding direct output please add a method to do the output computation.

Reply:
Done!